main/src/cgeo/geocaching/loaders/CoordsGeocacheListLoader.java

```diff
@@ -25,9 +26,12 @@ public GeocacheFilterType getFilterType() {
     @Override
     public IGeocacheFilter getAdditionalFilterParameter() {
         final DistanceGeocacheFilter distanceFilter = (DistanceGeocacheFilter) GeocacheFilterType.DISTANCE.create();
+        final int nearbySearchLimit = Settings.getNearbySearchLimit();
```

Comment:
The CoordsGeocacheListLoader is used for Nerby-Search, Coord-Search and Address-Search. My understanding was that the distance limitation should only apply for Nearby-Search. Thus this must be differentiated here.

Reply:
Lets overthink this a bit.
At least the problem of "overcoverage" in less cache densed areas also applies to coord/address-search...mmmh.

Reply:
> Lets overthink this a bit.
> At least the problem of "overcoverage" in less cache densed areas also applies to coord/address-search...mmmh.

I always thought that this is specific to "nearby" searches - since it is done in the field to find out what can immediately be reached. The other searches (by coordinate, address or keyword/name) seem more like something done during the planning phase of e.g. a cache tour. Here I would expect no distance limited results.

Also what then about the other searches (by keyword, by owner, found by)?

In any case, the current issue was specific to nearby search, so if this is really a discussion topic then a new issue should be created for it

Reply:
I agree to those arguments

additionally see https://github.com/cgeo/cgeo/issues/12451#issuecomment-1013762860

Reply:
> Lets overthink this a bit. At least the problem of "overcoverage" in less cache densed areas also applies to coord/address-search...mmmh.

Yes, there is the same problem.
If I want to search for a address (or coord), then I want to see caches around this place.
As sample: search for "Spiekeroog" for a holiday trip. The map of the result is:
![20220116_090416](https://user-images.githubusercontent.com/32555742/149670931-0a67f67c-ec5b-43c2-981a-9e96ff93cba3.jpg)
This is nearly unuseable, because I can not find the correct place. And a do not need caches in Hamburg.
A search by distance would be better here, then the destination will be placed nearly in the middle of the map.
A reduced amount of caches is fine, as loang as it is easy to reload more caches. (e.g. by switching to "live mode").